### PR TITLE
[*] PDF : Update TCPDF

### PR DESCRIPTION
**Pull request by firstred**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Add TCPDF to `composer.json` and update to version `6.2.12` |
| Type? | improvement |
| Category? | PDF |
| BC breaks? | no |
| Deprecations? | no |
| How to test? | Generate invoice, delivery slip or anything that touches `PDFGenerator.php`. |

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -120,8 +120,8 @@ GitHub contributors:
 - Florian Kwakkenbos
 - fram
 - Francois Gaillard
-- Francois-Marie de Jouvencel
-- Fran�ois Gaillard
+- François-Marie de Jouvencel
+- François Gaillard
```
Comment (firstred) [inline CONTRIBUTORS.md]: Spot the difference and win a free pull request!

Comment (firstred) [inline CONTRIBUTORS.md]: Spot the difference and win a free pull request!

**Comment by mickaelandrieu** [comment]
Sounds good to me (every move of library from ours to the packagist's one is good to me ^^).

Let's wait for QA for multiple tests and GO !

**Comment by Shudrum** [comment]
Ok, I've done some tests and I also want to remove all the old libraries imports.
As our QA team have 10 years of work to do in one week, I pretty sure I've missed many PDFs…

I hope that the old team have not modified this library, but I think not… so lets go!

**Comment by Quetzacoalt91** [comment]
It seems this PR also fixes another bug, thrown when a `tcpdf` folder is not yet generated in the cache.

Thanks

**Pull request merged**